Capri.Web/src/components/downBar.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+	<v-footer fixed height="128" style="" class="footerContainer">
+		<v-spacer></v-spacer>
+		<div class="footer">&copy; {{ new Date().getFullYear() }}</div>
+	</v-footer>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+
+@Component
+export default class DownBar extends Vue {}
+</script>
+<style lang="scss" scoped>
+.footerContainer {
+	left: 320px !important;
+	background-color: rgb(149, 149, 149) !important;
```

Comment:
Don't use `important`. Later you won't be able to override that. You can target specific class to override it.